.github/workflows/ci.yml

```diff
@@ -69,3 +68,32 @@ jobs:
     - name: Make sure `schema.graphql` is up to date
       working-directory: backend
       run: cargo run --bin export-schema | diff -u --color=always - ../frontend/src/schema.graphql
+
+    # Deployment
+    - name: Prepare binary and demo data for deployment
+      run: |
+        cp -v backend/target/release/tobira .deployment/files/
+        cp -v scripts/fixtures.sql .deployment/files/
+
+    - name: prepare deploy key
+      env:
+        DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+      run: |
+        install -dm 700 ~/.ssh/
+        echo "${DEPLOY_KEY}" > ~/.ssh/id_ed25519
+        chmod 600 ~/.ssh/id_ed25519
+        ssh-keyscan tobira.opencast.org >> ~/.ssh/known_hosts
+      if: ${{ github.event_name == 'push' }}
+
+    - name: install ansible postgres extensions
+      run: ansible-galaxy collection install community.general
+
+    - name: deploy tobira branch
+      working-directory: .deployment
+      run: >
+        ansible-playbook
+        --private-key=~/.ssh/id_ed25519
+        --extra-vars="branch='${GITHUB_REF#refs/heads/}'"
+        -u github
+        deploy.yml
```

Comment:
```suggestion
          --private-key=~/.ssh/id_ed25519
          --extra-vars="branch='${GITHUB_REF#refs/heads/}'"
          -u github
          deploy.yml
```

Maybe?

Reply:
Doing just that would break the deployment since it would convert the script to (note the newlines):

```
{'name': 'deploy tobira branch', 'working-directory': '.deployment', 'run': 'ansible-playbook\n  --private-key=~/.ssh/id_ed25519\n  --extra-vars="branch=\'${GITHUB_REF#refs/heads/}\'"\n  -u github\n  deploy.yml\n', 'if': "${{ github.event_name == 'push' }}"}
```

If we want to do something like that, we need to escape the linebreak.

Reply:
Oh ok. Nevermind then.